assets/src/edit-story/migration/migrate.js

```diff
@@ -22,12 +22,14 @@ import dataPixelTo1080 from './migrations/v0002_dataPixelTo1080';
 import fullbleedToFill from './migrations/v0003_fullbleedToFill';
 import squareToShape from './migrations/v0004_squareToShape';
 import mediaElementToResource from './migrations/v0005_mediaElementToResource';
+import setOpacity from './migrations/v0006_setOpacity';
```

Comment:
`v0005`? I think above `v0005` is a mistake. Should be already being fixed by Morten.